spec/rubocop/cop/layout/empty_lines_around_arguments_spec.rb

```diff
@@ -64,6 +64,29 @@
         .to eq(['Empty line detected around arguments.'])
     end
 
+    it 'on correct lines' do
```

Comment:
I think you probably wanted to use `context` here. After it `it` you should normally have some action. I notice the same for the previous example.

Reply:
http://www.betterspecs.org/#contexts

Reply:
Thanks! Fixed, and added one more example for good measure.